website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "v3.0.1",
```

Comment:
Does website/package.json#3 => version get a bump to 3.0.1 as well?

Reply:
@kkelly51 it can, but turns out it doesn't actually matter, and lerna isn't picking it up for .... reasons.    I can update it though.

Reply:
0d9fc9ec494b37483612b953df1d37d4b495d7cc